resources/js/components/progressBar.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+	<div style="width: 100%;">
+		
+		<span v-if="percentage==100">
+			<div class ="progress mt-4" style="width: 100%;">
+				<div v-bind:style="{width:percentage +'%'}" class="p-success progress-bar"></div>
+				<span style="position: absolute;right: 20px;">{{percentage}}% Complete</span>				
+			</div>
+		</span>
+		
+		<span v-if="percentage >49" class="m-0 p-0">
+			<span v-if="percentage<100" class="m-0 p-0">
+				<div class ="progress mt-4" style="width: 100%;">
+					<div v-bind:style="{width:percentage +'%'}" class="p-success progress-bar m-0"></div>
+					<span style="position: absolute;right: 20px;">{{percentage}}% Complete</span>				
+				</div>
+			</span>
+		</span>
+
+		<span v-if="percentage < 50">
+			<span v-if="percentage>0"> 				
+				<div class ="progress mt-4" style="width: 100%;">
+					<div v-bind:style="{width:percentage +'%'}" class="p-danger progress-bar"></div>
+					<span style="position: absolute;right: 20px;">{{percentage}}% Complete</span>				
```

Comment:
@saidabdul80 I thought we agreed on using Green(Success) progress bar regardless of the percentage.

Reply:
Sorry we didn't sir

Reply:
@abdullahikawu You did mention this during the last call? Or did I mistake it ?

Reply:
You're right Auwal, I discuss this with him over a call and he should change it in the subsequent commits.